src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -134,30 +136,44 @@ impl<S: S3BackEnd + Sync + Send + 'static> MetaData for S3MetaData<S> {
         &self.cache
     }
 
+    /// Set fuse fd into `MetaData`
+    async fn set_fuse_fd(&self, fuse_fd: RawFd) {
+        *self.fuse_fd.lock().await = fuse_fd;
+    }
+
     /// Delete node from trash
     async fn delete_trash(&self, ino: INum) -> bool {
```

Comment:
This function seems an attemption, shall we rename it to "try_delete_trash"?

Reply:
And please double check it's called everywhere that "lookup count" or "open count" is decreased including the local and remote triggers.

Reply:
Yes. everywhere has been checked. Will change to try_delete_trash. I think try_delete_node is better because we don't have trash anymore.

Reply:
Done